Misc/osx-build-wx

```diff
@@ -83,65 +125,144 @@ while getopts "dc:t:nk:t:" opt; do
 				carbon) TOOLKIT=CARBON;;
 				cocoa)  TOOLKIT=COCOA;;
 				*)      error Unexpected toolkit: $OPTARG. Its either \"carbon\" or \"cocoa\" \(default\) 1>&2
-						;;
+				;;
 			esac
 			;;	
-        k)
-            if [ -d $OPTARG ]
-            then
-                SDKPATH=$OPTARG
-            else
-				error Invalid directory passed for SDK path: $OPTARG
-            fi
-            ;;
-        t)
-            if [ -d $OPTARG ]
-            then
-                TOOLCHAIN_DIR=$OPTARG
-            else
-				error Invalid directory passed for toolchain dir: $OPTARG
-            fi
-            ;;
+		k)
+			if [ -d $OPTARG ]
+			then
+				SDKPATH=$OPTARG
+			else
+				error "Invalid directory passed for SDK path: $OPTARG"
```

Comment:
Should other error messages above be similarly quoted?

Reply:
Fair point.  Fixed.